spec/spec_helper.rb

```diff
@@ -22,17 +22,33 @@
 VCR.configure do |config|
   config.cassette_library_dir = 'tape_library'
   config.hook_into :webmock
+  # config.debug_logger = File.open('tape_library.log', 'w')
```

Comment:
Remove or move to comment if needed for future?

Reply:
Can be useful in the future, I moved it to the top and added a description.
Will be in the next commit.

Reply:
Move it to the top and added a description on then to use it. Will be in the next commit.